db/schema.rb

```diff
@@ -131,31 +136,17 @@
     t.index ["member_id", "group_id", "year"], name: "index_group_members_on_member_id_and_group_id_and_year", unique: true
   end
 
-  create_table "groups", id: :integer, force: :cascade do |t|
+  create_table "groups", id: :serial, force: :cascade do |t|
     t.string "name"
     t.integer "category"
-    t.text "comments", size: :medium
```

Comment:
Not sure if this migration step went okay, so please check

Reply:
Looks pretty wack but pretty sure it did